code/index.html

```diff
@@ -20,31 +20,53 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have...</h1>
 
       <select id="questions">
+        <!-- All of the HAIR options -->
         <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="brown">Brown hair?</option>
+          <option value="yellow">Yellow hair?</option>
+          <option value="black">Black hair?</option>
+          <option value="grey">Grey hair?</option>
+          <option value="orange">Orange hair?</option>
+          <option value="purple">Purple hair?</option>
+          <option value="white">White hair?</option>
+          <option value="hidden">Hidden hair?</option>
         </optgroup>
+
+        <!-- All of the EYES options -->
         <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+          <option value="blue">Blue eyes?</option>
+          <option value="brown">Brown eyes?</option>
+          <option value="green">Green eyes?</option>
+          <option value="hidden">Hidden eyes?</option>
         </optgroup>
+
+        <!-- All of the ACCESSORIES options -->
         <optgroup label="accessories">
-          <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="reading-glasses">Reading glasses?</option>
+          <option value="sun-glasses">Sun glasses?</option>
+          <option value="earrings">Earrings?</option>
+          <option value="necklace">A necklace?</option>
+          <option value="ring">A ring?</option>
+          <option value="hat">A hat?</option>
+          <option value="tie">A tie?</option>
+          <option value="eyepatch">An eyepatch?</option>
         </optgroup>
```

Comment:
Its nice that u added some more accessory then the original ones, good way to think outside the box :)
Looking on the pics and se what they really have on them.